specutils/io/default_loaders/pfs_spec.py

```diff
@@ -0,0 +1,80 @@
+"""
+Loader for PFS spectrum files.
+
+https://github.com/Subaru-PFS/datamodel/blob/master/datamodel.txt
+"""
+import os
+import re
+
+from astropy.io import fits
+from astropy.units import Unit
+from astropy.nddata import StdDevUncertainty
+
+import numpy as np
+
+from specutils.io.registers import data_loader
+from specutils import Spectrum1D
+
+__all__ = ['spec_identify', 'spec_loader']
+
```

Comment:
It would be great to get a comment here with a description of what this RE matching is doing.  Doesn't have to be super de-tailed, maybe just a couple examples of matching strings.